packages/otelbin-validation-image/src/index.ts

```diff
@@ -23,7 +24,102 @@ interface Env {
 
 const distroName = process.env.DISTRO_NAME;
 
-export const validateAdot = async (otelcolRealPath: string, configPath: string, env: Env): Promise<void> => {
+/**
+ * Using CommonJS export to allow for AWS lambda layer to work as per the
```

Comment:
large diff. Only thing changed here is the ESM -> CommonJS migration for auto-instrumentation support.